.travis.yml

```diff
@@ -0,0 +1,14 @@
+# references:	
+# * https://www.objc.io/issues/6-build-tools/travis-ci/	
+# * https://github.com/supermarin/xcpretty#usage	
+
+osx_image: xcode7.3	
+language: objective-c	
+# cache: cocoapods	
+# podfile: Example/Podfile	
+# before_install:	
+# - gem install cocoapods # Since Travis is not always on latest version	
+# - pod install --project-directory=Example	
+script:	
+- set -o pipefail && xcodebuild test -enableCodeCoverage YES -workspace Example/PatchKit.xcworkspace -scheme PatchKit-Example -sdk iphonesimulator9.3 ONLY_ACTIVE_ARCH=NO | xcpretty	
+- pod lib lint
```

Comment:
why did we add travis? ideally we integrate this into bitrise if we want to use CI but i guess we can do that sometime in the future @ywsang

Reply:
I mean it just runs a pod lib lint script that just makes sure no one breaks the pod setup i feel like we should keep it?

Reply:
Jk i see what ur saying. Ill remove